pkg/controller/clusters/status.go

```diff
@@ -68,6 +82,16 @@ func UpdateClusterStatusForCNI(ctx context.Context, cluster *anywherev1.Cluster)
 	}
 }
 
+// updateControlPlaneReadyConditionOnExternalETCD updates the ControlPlaneReady condition if etcdadm cluster is not ready.
+func updateControlPlaneReadyConditionOnExternalETCD(cluster *anywherev1.Cluster, kcp *controlplanev1.KubeadmControlPlane, etcdadmCluster *etcdv1.EtcdadmCluster) {
```

Comment:
nit: i think etcd is a name, not an acronym
```suggestion
func updateControlPlaneReadyConditionOnExternalEtcd(cluster *anywherev1.Cluster, kcp *controlplanev1.KubeadmControlPlane, etcdadmCluster *etcdv1.EtcdadmCluster) {
```